common/lib/xmodule/xmodule/tests/test_html_module.py

```diff
@@ -94,3 +94,15 @@ def test_index_dictionary(self):
             "content": {"html_content": " Text has spaces :) ", "display_name": "Text"},
             "content_type": "HTML Content"
         })
+
+        sample_xml_comment = '''
+            <html>
+                <p>This has HTML comment in it.</p>
```

Comment:
I'd also add a comment above the content, and one within - or more tests with those additional comments in place.